pino.js

```diff
@@ -170,7 +184,7 @@ function Pino (opts, stream) {
 }
 
 Pino.prototype = new EventEmitter()
-defineLevelsProperty(Pino.prototype)
+defineLevelsProperty(Pino.prototype, defaultLevelsValues)
```

Comment:
See my latest update to #168 . This is accomplished with less changes by simply adding the `levels` property at construction instead of on the prototype.